saleor/graphql/account/mutations.py

```diff
@@ -642,3 +659,58 @@ def mutate(cls, root, info, id, type):
 
         utils.change_user_default_address(user, address, address_type)
         return cls(errors=errors, user=user)
+
+
+class UserImageUpdate(BaseMutation):
+    user = graphene.Field(User, description='An updated user instance.')
+
+    class Arguments:
+        image = Upload(
+            required=True,
+            description='Represents an image file in a multipart request.',
+        )
+
+    class Meta:
+        description = dedent(
+            '''
+            Create a user image. This mutation must be
+            sent as a `multipart` request. More detailed specs of the upload
+            format can be found here:
+            https://github.com/jaydenseric/graphql-multipart-request-spec
+            '''
+        )
+
+    @classmethod
+    @staff_member_required
+    def mutate(cls, root, info, image):
+        user = info.context.user
+        errors = []
+        image_data = info.context.FILES.get(image)
+        validate_image_file(cls, image_data, 'image', errors)
+
+        if not errors:
+            if user.avatar:
+                user.avatar.delete_sized_images()
+                user.avatar.delete()
+            user.avatar = image_data
+            user.save()
+            create_user_avatar_thumbnails.delay(user_id=user.pk)
+
+        return UserImageUpdate(user=user, errors=errors)
+
+
+class UserImageDelete(BaseMutation):
```

Comment:
Similarly here we should use the name `UserAvatarDelete` (and update the mutation description as well).